### DISCUSSION TITLE
Please Add option for forking a repository or project.

### DISCUSSION THREAD

**Discussion by virendersran01** [community/community#3333] [Mobile]
Add forking option in mobile app.

**Comment by candyho** [comment]
Hey, thanks for your feedback. Could you tell us a little bit more on your use case - are you looking to fork the repository or project on mobile and then do editing on a desktop device or are you expecting to edit on mobile right after forking?

**Reply by TobiasRuano** [reply]
It would be nice to have some sort of editing option like on the website. Ex: be able to make quick changes to a README file from my phone, instead of going to the web.

**Reply by Crgster** [reply]
I would support this, the ability to Fork, edit and create a pull request from the mobile would be a great addition.

**Reply by jsoref** [reply]
Fwiw, I just wanted this feature because I wanted to test to see if GitHub mobile would tell me if a fork was a fork. https://github.com/community/community/discussions/34

---

For normal use cases, I'm probably going to edit a markdown file to create a PR or edit project metadata to show an upstream how the repository could be configured.

It would be sufficient to have an `open repository in browser` link/button to cover all features that aren't accessible from the mobile app, e.g. https://github.com/community/community/discussions/11095

**Reply by candyho** [reply]
We have added the ability to fork a public repo on mobile! 🙌 
https://github.blog/changelog/2024-12-11-now-you-can-fork-public-repositories-on-github-mobile/

**Comment by jsoref** [comment]
https://twitter.com/github/status/1866883185742622746 claims the feature exists

**Reply by candyho** [reply]
Yes, we have just shipped the feature on Tuesday! 🙌
